src/Swashbuckle.AspNetCore.SwaggerGen/SchemaGen/SchemaGenerator.cs

```diff
@@ -38,11 +38,20 @@ public SchemaGenerator(SchemaGeneratorOptions options, JsonSerializerSettings se
 
         public OpenApiSchema GenerateSchema(Type type, SchemaRepository schemaRepository)
         {
+            // Check if is nullable
+            var isNullable = type.IsNullable();
```

Comment:
Should be `isNullable = type.IsNullable() || type.IsFSharpOption`

Reply:
Fixed